src/libs/ReportActionsUtils.ts

```diff
@@ -701,7 +701,7 @@ function isReportActionAttachment(reportAction: OnyxEntry<ReportAction>): boolea
     const message = reportAction?.message?.[0];
 
     if (reportAction && ('isAttachment' in reportAction || 'attachmentInfo' in reportAction)) {
-        return Boolean(reportAction.isAttachment ?? reportAction.attachmentInfo ?? false);
+        return reportAction.isAttachment ?? !!reportAction.attachmentInfo ?? false;
```

Comment:
Should update to 
```
return reportAction?.isAttachment ?? reportAction?.attachmentInfo ?? false;
```

We only should check reportAction?.attachmentInfo if reportAction?.isAttachment is null or undefined

Reply:
@DylanDylann, sorry but what do you mean. I have used !! In attachmentInfo.

Reply:
Using reportAction?.isAttachment and reportAction?.attachmentInfo

Reply:
Ah, thanks for explanation. Let me change that.